include/picongpu/particles/ionization/byCollision/ThomasFermi/ThomasFermi_Impl.hpp

```diff
@@ -134,16 +134,19 @@ namespace ionization
              *  @todo Include all ion species because the model requires the
              *        density of ionic potential wells
              */
-            using DensitySolver = typename particleToGrid::CreateDensityOperation<T_SrcSpecies>::type::Solver;
+            using DensitySolver = typename particleToGrid::CreateFieldTmpOperation<
```

Comment:
If you are using `CreateFieldTmpOperation_t` here than the `::type::Solver` will be gone.

```C++
using DensitySolver = typename particleToGrid::CreateFieldTmpOperation_t<
    T_SrcSpecies,
￼     particleToGrid::derivedAttributes::Density
￼>::Solver;
```